lib/linux_admin/ssh_agent.rb

```diff
@@ -0,0 +1,94 @@
+require 'net/ssh'
+require 'fileutils'
+module LinuxAdmin
+  class SshAgent
+    attr_reader :pid
+    attr_reader :sock
```

Comment:
Please use the full word `socket` over `sock`.  No need for the abbreviation and it only leads to cognitive dissonance.